src/App.js

```diff
@@ -1,19 +1,45 @@
 import React from 'react';
 import './App.css';
-import chatMessages from './data/messages.json';
+import chatMessages from './data/messages.json'
+// import ChatEntry from './components/ChatEntry'
+import ChatLog from './components/ChatLog'
+import {useState} from 'react';
 
 const App = () => {
+  const [chatEntry, setChatEntry] = useState(chatMessages)
```

Comment:
Nice setup for your main `chatEntry` state variable and great work setting this at the very top of your component so it's easy to find!